### PR TITLE
add like method to take concrete or optional value

### PR DESCRIPTION
**Pull request by mikeclimbrock**

### CODE REVIEW

File: src/test/scala/io/flow/postgresql/QuerySpec.scala [hunk 1]
```diff
@@ -54,6 +54,22 @@ class QuerySpec extends FunSpec with Matchers {
     )
   }
 
+  it("likeOption") {
```
Comment (ericluria) [inline src/test/scala/io/flow/postgresql/QuerySpec.scala]: What happens when the parameter `None`?

File: src/test/scala/io/flow/postgresql/QuerySpec.scala [hunk 2]
```diff
@@ -54,6 +54,30 @@ class QuerySpec extends FunSpec with Matchers {
...
+    validate(
+      Query("select * from users").like("email", Some("mike")),
+      "select * from users where email like trim({email})",
+      "select * from users where email like trim('%mike%')"
```
Comment (ericluria) [inline src/test/scala/io/flow/postgresql/QuerySpec.scala]: image

**Comment by ericluria** [comment]
lgtm

File: src/test/scala/io/flow/postgresql/QuerySpec.scala (hunk 2 above)
Comment (mbryzek) [inline src/test/scala/io/flow/postgresql/QuerySpec.scala]:
```
 where email like trim({email}
```

you'll want this to be case insensitive and note in psql like requires '%'

Comment (mikeclimbrock) [inline src/test/scala/io/flow/postgresql/QuerySpec.scala]: i will make it case insensitive but it does get interpolated as `%`

**Comment by mikeclimbrock** [comment]
ack, will move to `search`

**Pull request closed without merging**